pyomo/common/__init__.py

```diff
@@ -10,11 +10,18 @@
 
 # The log should be imported first so that the Pyomo LogHandler can be
 # set up as soon as possible
-import pyomo.common.log
+from . import log
```

Comment:
Can you "record" the reason for changing from absolute imports to relative? Is this a practice we should be doing everywhere? If so, why?

Reply:
Relative imports used to be discouraged back when the syntax was ambiguous (pre PEP 328).  Now they are unambiguous, they are permitted and used as a matter of personal style (see comment by GvR: https://mail.python.org/pipermail/python-dev/2010-October/104476.html).

I moved to it because @santiagoropb had started using them and I liked how it allowed gently package reorganization without editing a ton of old imports.